bullet_train-incoming_webhooks/test/controllers/webhooks/incoming/bullet_train_webhooks_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require "test_helper"
+
+class Webhooks::Incoming::BulletTrainWebhooksControllerTest < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
+
+  # We can only run this test when Scaffolding things are enabled
+  # because the default webhook triggers come from `CreativeConcept`, etc.
+  unless scaffolding_things_disabled?
```

Comment:
@gazayas, I think we don't want this conditional here, if at all possible. After scaffolding an incoming webhook I'd want the tests for the controller to run, even if I'd set `ENV['HIDE_THINGS']` to `true`. (I don't know much about how incoming webhooks work behind the scenes, so apologies if this isn't really possible. Or easy.)

Reply:
@jagthedrummer After looking through the code for a bit and seeing what type of event types we have, I could probably pull this off with a Membership or Invitation. Will try to see what I can do.